specification/compute/resource-manager/readme.md

```diff
@@ -207,6 +215,10 @@ directive:
       - TrackedResourcePatchOperation
     reason:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
+  - where:
+      - $.definitions.RestorePoint
```

Comment:
Why is this needed?

Reply:
I requested and was granted approval for this suppression. Without it, the automated Linter fails.

Person Requesting Approval: roip@microsoft.com Submission Time: 3/11/2021 9:06:41 PM Link to PR: https://github.com/Azure/azure-rest-api-specs/pull/13219 Summary of the violations: R4015 - NestedResourcesMustHaveListOperation. The nested resource 'RestorePoint' does not have list operation, please add it.

Justification for supression: My team is not going to support this list operation for this new RestorePoint resource.

Reply:
I will add a reason in this PR (in the reason section)